.github/workflows/test.yml

```
@@ -5,11 +5,11 @@ on: [push, pull_request]
 jobs:
   js-bootstrap:
     name: Test JS bootstrap on Node v${{ matrix.node-version }}
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-20.04
 
     strategy:
       matrix:
-        node-version: [ 12.x, 14.x, 16.x ]
+        node-version: [ 21.x, 22.x, 23.x ]
```

It's better to test even numbered node versions, because they're the ones that are supported for production. I suggest using 

```suggestion
        node-version: [ 18.x, 20.x, 22.x ]
```